tests/integration/targets/filesystem/defaults/main.yml

```diff
@@ -16,7 +16,7 @@ tested_filesystems:
   ext3: {fssize: 10, grow: True}
   ext2: {fssize: 10, grow: True}
   xfs: {fssize: 20, grow: False}  # grow requires a mounted filesystem
-  btrfs: {fssize: 150, grow: False}  # grow not implemented
+  btrfs: {fssize: 150, grow: False}  # grow requires a mounted filesystem
```

Comment:
Have you tried setting `grow` to `True` and see what happens? Might be worth a try :)

Reply:
As suspected, it doesn't work:
```
TASK [filesystem : Expand filesystem] *********************************************************************************************************************************************
fatal: [testhost]: FAILED! => {"changed": false, "msg": "/tmp/ansible.lal4h9yf.test/img needs to be mounted for Btrfs operations"}
```

Reply:
Good question. If you're interested in trying to change that, feel free to create a new PR for that once this is merged. (Doing that in this PR is probably a bit too much.)